packages/manager/modules/server-sidebar/src/controller.js

```diff
@@ -60,6 +61,10 @@ export default class OvhManagerServerSidebarController {
             this.SIDEBAR_ORDER_CONFIG = DEDICATED_ORDER_SIDEBAR_CONFIG;
           }
 
+          if (this.universe === 'WEB' && find(universes, { universe: this.universe.toLowerCase() })) {
+            this.SIDEBAR_CONFIG = WEB_SIDEBAR_CONFIG;
+            this.SIDEBAR_ORDER_CONFIG = WEB_ORDER_SIDEBAR_CONFIG;
+          }
```

Comment:
Could we imagine generating this dynamically ? I had in mind something like 
```suggestion
          if (find(universes, { universe: this.universe.toLowerCase() }))) {
           this.SIDEBAR_CONFIG = [universe]_SIDEBAR_CONFIG
           }
```
but not sure it is quite possible though 
That way we don't have to wonder about adding a `if this.universe ===`

Reply:
Thanks for pointing out this part.

You're right! It could be addressed in separate pull request since a new release is required.